diff --git i/cmake/DownloadProject.CMakeLists.cmake.in w/cmake/DownloadProject.CMakeLists.cmake.in
index 6ab48614..a242c9d5 100644
--- i/cmake/DownloadProject.CMakeLists.cmake.in
+++ w/cmake/DownloadProject.CMakeLists.cmake.in
@@ -1,7 +1,7 @@
 # Distributed under the OSI-approved MIT License.  See accompanying
 # file LICENSE or https://github.com/Crascit/DownloadProject for details.
 
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.5)
 
 project(${DL_ARGS_PROJ}-download NONE)
 
diff --git i/meshfix/JMeshExt-1.0alpha_src/OpenNL3.2.1/CMakeLists.txt w/meshfix/JMeshExt-1.0alpha_src/OpenNL3.2.1/CMakeLists.txt
index f6f5365e..24b904a1 100755
--- i/meshfix/JMeshExt-1.0alpha_src/OpenNL3.2.1/CMakeLists.txt
+++ w/meshfix/JMeshExt-1.0alpha_src/OpenNL3.2.1/CMakeLists.txt
@@ -8,13 +8,13 @@ include_directories(SuperLU/SRC)
 
 if(COMMAND cmake_policy)
 	if(POLICY CMP0003)
-		cmake_policy(SET CMP0003 OLD)
+		cmake_policy(SET CMP0003 NEW)
 	endif(POLICY CMP0003)
 	if(POLICY CMP0005)
-		cmake_policy(SET CMP0005 OLD)
+		cmake_policy(SET CMP0005 NEW)
 	endif(POLICY CMP0005)
 	if(POLICY CMP0011)
-		cmake_policy(SET CMP0011 OLD) # or NEW
+		cmake_policy(SET CMP0011 NEW) # or NEW
 	endif(POLICY CMP0011)
 endif(COMMAND cmake_policy)
 
